### PR TITLE
Add Ansible specific templates

### PR DESCRIPTION
**Pull request by KB-perByte**
## What does this PR do / why we need it
Add Ansible scaffolder-backend-plugin specific templates
- adds a collection scaffolding template.
- adds a playbook scaffolding template.

expected app-config configuration -
```
ansible:
  creatorService:
    baseUrl: 127.0.0.1
    port: '5000'
  devSpacesBaseUrl: >-
    https://devspaces.dummy.test-tests.testing.check.com/
```

## Which issue(s) does this PR fix
- NA
Fixes #?

## How to test changes / Special notes to the reviewer
- Dependent on the ansible-backend plugin

### CODE REVIEW

File: templates/ansible/skeletons/catalog-info/catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+apiVersion: backstage.io/v1alpha1
```
Comment (KB-perByte) [inline templates/ansible/skeletons/catalog-info/catalog-info.yaml:1]: Shall we keep it within the ansible dir, or move it to the common skeletons dir?

File: templates/ansible/playbook/template.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        collectionGroup: '${{ parameters.collectionGroup }}'
+        collectionName: '${{ parameters.collectionName }}'
+        applicationType: playbook-project
+        sourceControl: github.com
+
+    - id: catalogTemplate
+      name: Generating the Catalog Info Component
+      action: fetch:template
+      input:
+        url: ../skeletons/catalog-info/
```
Comment (KB-perByte) [inline templates/ansible/playbook/template.yaml:76]: should this be, `../../../skeletons/ansible/catalog-info` ? the `ansible/skeletons/catalog-info` to be moved to common skeletons, then!

File: templates/ansible/skeletons/catalog-info/catalog-info.yaml (hunk 1 above)
Comment (kadel) [reply templates/ansible/skeletons/catalog-info/catalog-info.yaml:1]: Would it be possible to re-use the existing skeleton for catalog-info.yaml? Having one centralized catalog-info template is going to make maintenance much easier.

You can extend existing https://github.com/redhat-developer/red-hat-developer-hub-software-templates/blob/main/skeletons/catalog-info/catalog-info.yaml to add things that you need

**Review by kadel — CHANGES_REQUESTED** [review]

**Comment by kadel** [comment]
@KB-perByte can you point me to some docs that explain how to deploy creatorService? Or is there somewhere hosted instance that I can use to test this?

**Comment by KB-perByte** [comment]
@kadel we are closing this PR as we planned to keep the templates in a separate repo, which makes the templates independent of RHDH releases.
As the ansible's backend plugin is dependent on the template it would be easy for us to maintain version-specific changes in the template with sync to plugin changes.
Please let us know if this is not the most advised process RHDH standards.
Regards.

**Pull request closed without merging**